aQute.libg/build.gradle

```diff
@@ -0,0 +1,3 @@
+test {
+    useJUnitPlatform()
```

Comment:
I don't like this. We do NOT want to put build.gradle files in every project. Anything we need to do for gradle for Junit 5 needs to be handled centrally in the root build.gradle file.

Reply:
I didn't want to go big bang on all subprojects. I suppose it can still be added selectively from the root build.gradleI. The ideal solution, IMO, as I mentioned, is for the Gradle plugin to add it when it notices bnd.bnd has specified Junit Platform on the classpath.

Reply:
I've changed it so that all projects are now using junit-platform.